index.html

```diff
@@ -231,11 +231,13 @@
 
 # Thank you
 
-Meili
```

Comment:
Hello Meili,
I'm KANKINDI Ritha Veneranda and I really appreciate you for an interesting course you gave us today. it's a great contribution to my job searching journey and also empowering my career. be blessed!